plugins/node/opentelemetry-instrumentation-pino/package.json

```diff
@@ -58,16 +64,16 @@
     "gts": "3.1.0",
     "mocha": "7.2.0",
     "nyc": "15.1.0",
+    "pino": ">=8.0.0",
     "rimraf": "3.0.2",
+    "semver": "^7.3.5",
```

Comment:
usually we pin devDeps (except otel core components) to avoid "auto breaking" CI because of new releases

Reply:
https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1108/commits/8b6f0b76fd741b577d80078b623caeabe73df26c